README.md

```diff
@@ -44,8 +44,8 @@ None.
 Here's how to use it in a playbook:
 
 ```yaml
-- hosts: web
-  become: yes
+- hosts: all
```

Comment:
Was this change intentional?

```suggestion
- hosts: web
```

Reply:
Yes.  It doesn't matter and hence I figured it may as well match the skeleton.